python/tests/plugins/validators/test_validation_implementation_plugin.py

```diff
@@ -27,14 +31,18 @@ def test_validation_module_register_validators(self):
     def test_validate_validator_implementations_validate_validation_success(self):
         test_active_context = get_active_context(reload_context=True)
 
-        validation_definitions = list(filter(lambda definition: definition.get_root_key() == "validation", test_active_context.definitions))
+        validation_definitions = list(
+            filter(lambda definition: definition.get_root_key() == "validation", test_active_context.definitions)
+        )
```

Comment:
I'd also update to use the constants, but I won't nit pick that on a large PR like this.
```suggestion
        validation_definitions = test_active_context.get_definitions_by_root_key("validation")
```